docs/host-and-deploy/hosting-models.md

```diff
@@ -45,13 +45,27 @@ The downsides to the client-side hosting model are:
 * Larger download size and app load time
 * Less mature .NET runtime and tooling support (for example, limitations in .NET Standard support and debugging)
 
+> [!NOTE]
+> Visual Studio includes the **Blazor (ASP.NET Core hosted)** project template for creating a Blazor app that runs on WebAssembly and is hosted on an ASP.NET Core server. The ASP.NET Core app serves the Blazor app to clients but is otherwise a separate process. The client-side Blazor app can interact with the server over the network using Web API calls or SignalR connections.
+
+> [!IMPORTANT]
+> If a Blazor app (based on the **Blazor** project template) uses the client-side (in-process) hosting model and is hosted as an IIS sub-application in an ASP.NET Core app, it's important to disable the inherited ASP.NET Core Module handler. Remove the handler in the Blazor app's published *web.config* file by adding a `<handlers>` section to the file:
```

Comment:
> If a Blazor app (based on the **Blazor** project template) uses the client-side (in-process) hosting model and is hosted as an IIS sub-application in an ASP.NET Core app

I had to read this a bunch of times to figure out what it was saying. Is there a way to simplify this text? How about:

If a client-side Blazor app is hosted in an ASP.NET Core app that is hosted in an IIS sub-application, ...